consistency with Identity Provider dropdown

spottedmahn opened the pull request:
the identity dropdown for local accounts has 'email' and 'username'.

also, in the same sentence 'sign up' and 'signup' were used.  made them consistent

azurecla commented:
Hi __@shuainie__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (shuainie). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

spottedmahn committed:
consistency with Identity Provider dropdown

the identity dropdown for local accounts has 'email' and 'username'.

also, in the same sentence 'sign up' and 'signup' were used.  made them consistent
